config/initializers/recaptcha.rb

```diff
@@ -0,0 +1,4 @@
+Recaptcha.configure do |config|
+  config.site_key  = '6LdvVj8UAAAAAPgGvyqQRFWa6Kq1xW2h-h_Re8sT'
```

Comment:
do not store config key
use `.env`
THIS IS CRITICAL

Reply:
Fixed